### ISSUE TITLE
List Metastore Assignments does not work

### ISSUE THREAD

**Issue by andy-potter-evotix** [databricks/databricks-sdk-py#321]
The metastore_assignments.list(metastore_id) method of the account client does not work. It fails with an exception trying to process the response JSON because the response does not match the format expected. The SDK attempts to build an array of MetastoreAssignment objects but the response JSON from the Databricks API is just an array of workspace_ids.

**Comment by andy-potter-evotix** [comment]
I don't have a code sample but I can tell you we are using the 'list' method of the metastore_assignements API of the AccountClient. We've worked around the problem for now by creating our own method which uses the _api.do method of the metastore_assignements API:

```
def list_metastore_assignments(account_client: AccountClient, metastore_id: str) -> Iterator[int]:
    headers = {'Accept': 'application/json', }
    res = account_client.metastore_assignments._api.do('GET',
                        f'/api/2.0/accounts/{account_client.metastore_assignments._api.account_id}/metastores/{metastore_id}/workspaces',
                        headers=headers)
    return [v for v in res.get('workspace_ids', [])]
```

**Comment by mgyucht** [comment]
@andy-potter-evotix This is fixed as of version 0.10.0. https://github.com/databricks/databricks-sdk-py/commit/bbe81c13c4a0275020d241b78f61cdddf349adc2#diff-c218f52b5f12f62b62eec865d36889e5d7cc7beb63122e8026658093de3283d0R1532

Thank you for reporting this!
